Chapter15/peer.erl

```diff
@@ -0,0 +1,86 @@
+-module(peer).
+-export([start/0, connect/1, send/1, stop/0]).
+-export([wait_connect/1]).
+-export([loop/1]).
+
+% Exercise 15-3: Peer to Peer
+% I couldent figure out how to get the crypto module working for encryption
```

Comment:
there's docs about crypto: https://github.com/joearms/crypto_tutorial

Reply:
I figured out how to implement simple encryption using hardcoded keys, if you would like me to implement a better solution (not using hardcoded keys) let me know and I'll get it updated.